include/eigensolve_quda.h

```diff
@@ -339,8 +372,50 @@ namespace quda
        @param[in] file The filename to save
     */
     void loadFromFile(const DiracMatrix &mat, std::vector<ColorSpinorField *> &eig_vecs, std::vector<Complex> &evals);
+
+    /**
+       @brief Sort array the first n elements of x according to spec_type, y comes alond for the ride 
```

Comment:
typo "alond" -> "along", propagates for next three functions.

We should unify these routines---there should be some way to do it with templates, traits, and some `std::enable_if` magic. I can investigate it, let me know if that'd help @cpviolator .

Reply:
Thinking about this...